utilities/src/main/java/io/syndesis/qe/TestConfiguration.java

```diff
@@ -151,6 +151,25 @@ public static String openShiftSARNamespace() {
     }
 
     public static String openShiftRouteSuffix() {
+        if (get().readValue(OPENSHIFT_ROUTE_SUFFIX) == null) {
+            final String prefix = "apps.";
+            if (get().readValue(SYNDESIS_UI_URL) != null && get().readValue(SYNDESIS_UI_URL).contains("syndesis.my-minishift.syndesis.io")) {
+                //for jenkins
+                get().overrideProperty(OPENSHIFT_ROUTE_SUFFIX, "my-minishift.syndesis.io");
+            } else if (openShiftUrl().endsWith("8443")) {
+                //OCP 3.11
+                if (openShiftUrl().matches("https:\\/\\/\\d{1,4}.\\d{1,4}.\\d{1,4}.\\d{1,4}:8443")) {
```

Comment:
this can be a little bit simplified to `"https:\\/\\/(\\d{1,4}\\.){3}\\d{1,4}:8443"`